pkg/scraper/bruneida_scraper.go

```diff
@@ -0,0 +1,82 @@
+package scraper
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/b-open/jobbuzz/pkg/types"
+	"github.com/gocolly/colly"
+)
+
+func ScrapeBruneida() types.Jobs {
+	allowedDomain := colly.AllowedDomains("www.bruneida.com")
+	linkCollector := colly.NewCollector(
+		allowedDomain,
+	)
+	jobCollector := colly.NewCollector(
+		allowedDomain,
+	)
+
+	// links := make(links, 0)
+	jobs := make(types.Jobs, 0)
```

Comment:
Not necessary to fix but alternatively there is 2 different way to declare a slice if you dont need the length.

```go
jobs := []types.Jobs{}
var jobs []types.Jobs
```

Reply:
should be
```go
jobs := types.Jobs{}
```
Since `Jobs` is already `[]Job`